modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -56,163 +52,166 @@ For example:
-.. Using your web browser, navigate to the *Applications* tab in Runtime Manager.
-.. Select an application and select *Manage Application*. 
-.. The value in the *Runtime version* field indicates if an update is available. Select the updated version of Mule runtime engine.
-.. Select *Deploy* to redeploy the application with the updated Mule runtime engine version.
-.. Repeat this for each application running on this Runtime Fabric.
+To upgrade the Runtime Fabric appliance:
 
-. Verify Runtime Fabric is healthy:
-.. Using your web browser, navigate to the *Runtime Fabrics* tab in Runtime Manager and verify that Runtime Fabric reports an *Active* status.
-.. Open a terminal to a Runtime Fabric controller node.
-.. Run `/opt/anypoint/runtimefabric/rtfctl status` and confirm that the cluster status is healthy.
-.. Keep your terminal open to use in later steps.
-
-. Get the latest version of the Runtime Fabric appliance package:
-.. Using your web browser, navigate to Runtime Manager and select the *Runtime Fabrics* tab to view a list of Runtime Fabrics.
-.. Select the *Downloads* link.
-.. Select the *Copy Link* icon next to *Installer package* to copy the code snippet, as shown in the following example:
+. Using your web browser, navigate to Runtime Manager and select the *Runtime Fabrics* tab to view a list of Runtime Fabrics.
+. Select the Runtime Fabric you want to upgrade.
+. Next to the Appliance version, click *Upgrade Available*.
+. Select the *Copy Link* icon next to *Installer package* to copy the code snippet, as shown in the following example:
 +
 ```
 rtfctl appliance upgrade --url https://<upgrade-package-information>
 ```
-.. Paste the code snippet in a text editor for reference. 
+. Paste the code snippet in a text editor for reference. 
+
+=== Update the Mule Deployments Running on Runtime Fabric to the Latest Patch Version
+
+. Using your web browser, navigate to the *Applications* tab in Runtime Manager.
+. Select an application and select *Manage Application*. 
+. The value in the *Runtime version* field indicates if an update is available. Select the updated version of Mule runtime engine.
+. Select *Deploy* to redeploy the application with the updated Mule runtime engine version.
+. Repeat this for each application running on this Runtime Fabric.
+
+=== Verify Runtime Fabric is healthy
+. Using your web browser, navigate to the *Runtime Fabrics* tab in Runtime Manager and verify that Runtime Fabric reports an *Active* status.
+. Open a terminal to a Runtime Fabric controller node.
+. Run `/opt/anypoint/runtimefabric/rtfctl status` and confirm that the cluster status is healthy.
+. Keep your terminal open to use in later steps.
+
+=== Run the Upgrade
+
+The upgrade can take more than an hour to complete.
+
+The connection between Runtime Fabric and Runtime Manager can disconnect multiple times during the upgrade procedure. Connectivity is restored after the upgrade finishes.
```

Comment:
I wonder if we should tweak the wording to include what the audience should do about this, like have the free time or not worry?